Makefile

```
@@ -0,0 +1,121 @@
+# Export environment variables
+export DEV_PORT
+export DEV_HOST
+export NODE_ENV
+
+# Colors for output
+GREEN := \033[0;32m
+YELLOW := \033[0;33m
+RED := \033[0;31m
+NC := \033[0m # No Color
+
+.PHONY: help install dev dev-custom build build-all clean dist test lint format docker-build docker-run docker-dev
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)